vm/src/builtins/complex.rs

```diff
@@ -228,9 +228,15 @@ impl PyComplex {
     }
 
     #[pymethod(magic)]
-    fn abs(&self) -> f64 {
+    fn abs(&self, vm: &VirtualMachine) -> PyResult<PyArithmeticValue<f64>> {
```

Comment:
```suggestion
    fn abs(&self, vm: &VirtualMachine) -> PyResult<f64> {
```
Because `PyArithmeticValue::NotImplemented` is not used, `PyResult<f64>` will be enough.

Reply:
Oh yeah, you're right. I just fixed and pushed it again 👍